tests/MongoDB.Driver.Tests/AggregateFluentTests.cs

```diff
@@ -196,6 +196,55 @@ public void ChangeStream_should_add_the_expected_stage_when_options_is_null()
             result.Should().Be(1);
         }
 
+        [SkippableFact]
+        public void Group_with_accumulator_should_return_expected_result()
+        {
+            RequireServer.Check().Supports(Feature.AggregateAccumulator);
+
+            var client = CreateClient();
```

Comment:
I'm surprised we don't use `DriverTestConfiguration.Client` but this line matches other tests in this file.